src/routes/secured/theater.ts

```diff
@@ -0,0 +1,26 @@
+import express, { Response, Request } from 'express';
+
+import ResponseValue from '../../utils/ResponseValue';
+import { asyncHandler } from '../../middleware/async-handler';
+import { MovnetError } from '../../middleware/MovnetError';
+import { TheaterService } from '../../services/theater';
+
+module.exports = (router: express.Router) => {
+  /**
+  * Adds a new chat
+  */
+  router.post('/theater', asyncHandler(async (req: Request, res: Response) => {
+    const name = req.body.name;
+    const address = req.body.address;
+    const latitude = req.body.latitude;
+    const longitude = req.body.longitude;
+
+    if(!name || !address){
```

Comment:
`latitude` and `longitude` should be checked as well

Reply:
The reason that I wanted to leave it is because we don't have an easy way of finding this information. `cineplex-scraper` can only get city & province, so we would need another API which can convert this into longitude & latitude. What may be easier but also practical is to use city & province instead of longitude & latitude, since the routing platform we use for maps likely supports that form too.

Ex.
```
"address": "1680 Richmond Street",
"city": "London",
"province'': "Ontario"
```

instead of 
```
"address": "1680 Richmond Street",
"longitude": -81.2781354,
"latitude": 43.0269201
```